src/app/utils/pool-utils.ts

```diff
@@ -19,19 +19,19 @@ export interface PoolPriceOptions {
 }
 
 const iconMapping = {
-    "UbuntuServer": Icons.ubuntu,
+    "ubuntuserver": Icons.ubuntu,
     "ubuntu-server-container": Icons.ubuntu,
     "ubuntu-server-container-rdma": Icons.ubuntu,
-    "CentOS": Icons.centos,
-    "CentOS-HPC": Icons.centos,
-    "WindowsServer": Icons.windows,
-    "Debian": Icons.debian,
-    "Oracle-Linux": Icons.oracle,
+    "centos": Icons.centos,
+    "centos-hpc": Icons.centos,
+    "windowsserver": Icons.windows,
+    "debian": Icons.debian,
+    "oracle-linux": Icons.oracle,
```

Comment:
nice some consistency :)

Reply:
Yea reduce likely hood of accidental error